CMakeLists.txt

```diff
@@ -5,12 +5,12 @@ find_package(catkin REQUIRED COMPONENTS cmake_modules)
 find_package(Boost REQUIRED COMPONENTS filesystem program_options system)
 set(Python_ADDITIONAL_VERSIONS "${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}")
 find_package(PythonLibs "${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}" REQUIRED)
-find_package(TinyXML REQUIRED)
+find_package(TinyXML2 REQUIRED)
```

Comment:
The CI jobs fail since the Ubuntu package of `TinyXML2` doesn't provide a CMake module. This will likely require a custom module - same as for TinyXML - see https://github.com/ros/cmake_modules/blob/0.4-devel/cmake/Modules/FindTinyXML.cmake. Can you please provide a PR for this.